apps/dashboard/playwright.config.ts

```diff
@@ -38,9 +51,22 @@ export default defineConfig({
   },
   /* Configure projects for major browsers */
   projects: [
+    {
+      name: 'setup',
+      testMatch: /global\.setup\.ts/,
+    },
```

Comment:
The global setup test file will initialize the Clerk session for all tests. Using this approach, we can say that for some tests, we don't want to have a session, and then we will be able to verify, for example, the sign-in/up flows.